### PR TITLE
checkout session modification

### PR DESCRIPTION
**Pull request by plisek1986**

### CODE REVIEW

File: payments/views.py [hunk 1]
```diff
@@ -1,75 +1,63 @@
...
-from payments import schemas as pay_schemas
-from urllib.parse import urljoin
+from django.views.generic import FormView, TemplateView
 from users.models import Profile
-from payments.models import TopUp
-import stripe
+
+from .forms import TopUpForm
+from .models import TopUp
+from .schemas import (LineItems, LineItemsSchema, Metadata, MetadataSchema,
```
Comment (halypa) [inline payments/views.py]: Please use style:
```
from .schemas import (
   LineItems, 
   LineItemsSchema,
   Metadata, 
   MetadataSchema,
)
```

File: payments/views.py [hunk 2]
```diff
@@ -180,7 +178,8 @@ def increase_balance(event_body, topup):
 @method_decorator(login_required, name='dispatch')
 class PaymentHistoryView(View):
     def get(self, request):
-        user_payments = TopUp.payments.filter(user=request.user).filter(~Q(payment_intent_status='')).order_by('-date_created')
+        user_payments = TopUp.payments.filter(user=request.user).filter(~Q(payment_intent_status='')).order_by(
```
Comment (halypa) [inline payments/views.py]: `TopUp.payments.filter(user=request.user).exclude(payment_intent_status='').order_.....`

File: payments/views.py [hunk 3]
```diff
@@ -112,34 +103,37 @@ def post(self, request, *args, **kwargs):
...
 def get_transaction_record(event_body):
     topup_pk = event_body.metadata.topup_pk
     try:
-        topup = pay_models.TopUp.payments.get(pk=topup_pk)
+        topup = TopUp.payments.get(pk=topup_pk)
         return topup
-    except pay_models.TopUp.DoesNotExist:
-        print('record with this pk does not exist') #TODO handle error
+    except TopUp.DoesNotExist:
+        print('record with this pk does not exist')  # TODO handle error
```
Comment (halypa) [inline payments/views.py]: Print function is not recommended it this type application. Please use logger if is needed :D

File: payments/views.py (hunk 1 above)
Comment (plisek1986) [reply payments/views.py]: Done (with a bit of a delay, but done :) )

File: payments/views.py (hunk 2 above)
Comment (plisek1986) [reply payments/views.py]: Depreciated; modified to ListView

**Review by lemarta — APPROVED** [review]

**Pull request merged**